### PR TITLE
Submission details page test class addition

### PR DESCRIPTION
**Pull request by MadjarovaML**

### CODE REVIEW

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/class-names.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-type classNamesType = string | string[] | null | undefined;
+type classNamesType = string | string[] | null | undefined | {};
```
Comment (Minkov) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/utils/class-names.ts:1]: Does this work?
I think the result for `{"something": "otherthing"}` will be `[object Object]`

More code should be implemented in the join. I am not sure whether this logic should be here.

**Review by Minkov — CHANGES_REQUESTED** [review]

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/components/common/Props.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { ReactNode } from 'react';
 
-type ClassNameType = string | string[];
```
Comment (shsimeonova) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/components/common/Props.ts:3]: why is this needed?

File: automated_testing/test/tests/index-test.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { beforeEach } from 'mocha';
 import IndexPage from '../pageobjects/index-page';
 
-describe('Testing index', () => {
+describe.skip('Testing index', () => {
```
Comment (shsimeonova) [inline automated_testing/test/tests/index-test.ts]: I believe we dont want this in

File: automated_testing/test/tests/submissions-details-test.ts [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+        await SubmissionsDetailsPage.open();
+    });
+
+    after(async () => {
+        await AuthPage.performLogOut();
+    });
+
+    it('Expect left navigation to be there', async () => {
+        await browser.waitUntil(
+            () => browser.execute(() => document.readyState === 'complete'),
```
Comment (shsimeonova) [inline automated_testing/test/tests/submissions-details-test.ts:32]: we can extract this into a function and define the failure message

**Comment by shsimeonova** [comment]
This PR contains changes for contest page tests which i believe should be a separate PR.

File: automated_testing/test/tests/submissions-details-test.ts (hunk 4 above)
Comment (MadjarovaML) [reply automated_testing/test/tests/submissions-details-test.ts:32]: This was witten by you and it was only fo sanity check and I will delete it when the time comes. Also we discussed yesterday that the focus of this PR is only the class change and the tests part are in it by mistake.
